### ISSUE TITLE
cluster-plugin conflict with google ClusterManager

### ISSUE THREAD

**Issue by Dkhusainov** [mapbox/mapbox-plugins-android#185]
When you have both yours and google cluster libraries in your apk, layouts from these libs are conflicting because they have the same names. This error happens when trying to create google ClusterManager:

```java
java.lang.ClassCastException: com.mapbox.mapboxsdk.plugins.cluster.ui.RotationLayout cannot be cast to com.google.maps.android.ui.RotationLayout
                                                                        at com.google.maps.android.ui.IconGenerator.<init>(IconGenerator.java:62)
                                                                        at com.google.maps.android.clustering.view.DefaultClusterRenderer.<init>(DefaultClusterRenderer.java:135)
                                                                        at com.google.maps.android.clustering.ClusterManager.<init>(ClusterManager.java:76)
                                                                        at com.google.maps.android.clustering.ClusterManager.<init>(ClusterManager.java:68)
```
I know it seems like a google issue, but it isn't. Because you named the layout's with identical names

sample project: https://github.com/Dkhusainov/mapbox-cluster-error

**Comment by goldinnikolai** [comment]
+1

**Comment by langsmith** [comment]
cc @tobrun 

Thanks @Dkhusainov ; good catch. We will look into it

**Comment by tobrun** [comment]
When porting the Google library we didn't change the prefix of the resources. 
For all resources we need to change the `amu` prefix to something different:
> amu_info_window.xml -> mbc_info_window.xml

I wouldn't use `mapbox_` as this could conflict with the maps SDK.

**Comment by langsmith** [comment]
@Dkhusainov & @goldinnikolai 

This issue has been resolved via #192 .  Thanks for alerting us to this and for using Mapbox!

**Comment by GitHubMurt** [comment]
I'd to reopen issue

line below causes exception
`clusterManager = ClusterManagerPlugin(this, mapbox)`

> Exception in MapView.OnMapChangedListener
>                                                                java.lang.ClassCastException: com.google.maps.android.ui.RotationLayout cannot be cast to com.mapbox.mapboxsdk.plugins.cluster.ui.RotationLayout
>                                                                    at com.mapbox.mapboxsdk.plugins.cluster.ui.IconGenerator.<init>(IconGenerator.java:51)
>                                                                    at com.mapbox.mapboxsdk.plugins.cluster.clustering.view.DefaultClusterRenderer.<init>(DefaultClusterRenderer.java:123)
>                                                                    at com.mapbox.mapboxsdk.plugins.cluster.clustering.ClusterManagerPlugin.<init>(ClusterManagerPlugin.java:68)
>                                                                    at com.mapbox.mapboxsdk.plugins.cluster.clustering.ClusterManagerPlugin.<init>(ClusterManagerPlugin.java:60)
>                                                                    at pl.intenso.carefleet.UI.mapPOI.experimental.GasStationMapBox$onCreate$1.onMapReady(GasStationMapBox.kt:51)
>                                                                    at com.mapbox.mapboxsdk.maps.MapView$MapCallback.onMapReady(MapView.java:1060)
>                                                                    at com.mapbox.mapboxsdk.maps.MapView$MapCallback.onMapChanged(MapView.java:1045)
>                                                                    at com.mapbox.mapboxsdk.maps.MapView.onMapChange(MapView.java:582)
>                                                                    at com.mapbox.mapboxsdk.maps.NativeMapView.onMapChanged(NativeMapView.java:862)
>                                                                    at android.os.MessageQueue.nativePollOnce(Native Method)
>                                                                    at android.os.MessageQueue.next(MessageQueue.java:330)
>                                                                    at android.os.Looper.loop(Looper.java:137)
>                                                                    at android.app.ActivityThread.main(ActivityThread.java:5621)
>                                                                    at java.lang.reflect.Method.invoke(Native Method)
>                                                                    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:794)
>                                                                    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:684)
